test/integration/features/cmd-timezone.feature

```diff
@@ -0,0 +1,72 @@
+@cmd-timezone @core
+Feature: Minishift timezone subcommand
+Timezone subcommand is used for listing and applying timezone 
+availaible to Minishift running VM instance.
+
+@quick
```

Comment:
@amitkrout @agajdosi I do not think it is a quick test. We should think about how we can manage these tests with less `minishift start` across feature files (in long term) so that we can reduce the execution time.

Reply:
@LalatenduMohanty Right, but think about the total time taken here in ```@quick``` tag. It is a fraction of second right ? and the total time taken in this feature file will be max 1 sec. So IMO even in long run ```quick``` tag can be compromised as it cost very less time.

Reply:
@amitkrout We have a `minishift start` here, so not sure how it will takes seconds.

Reply:
@LalatenduMohanty In the feature file, place holder of the tag describes the coverage of the tag like tag define on top of the file act as a global tag for that file and tag those are just above any ```scenario:``` acts like local, covers only that scenario in the same feature file. 

So in ```cmd-timezone.feature``` only three quick scenario are available which does not cover any ```minishift start``` scenario. For clarification of how the local tag works for a specific feature file you can run the command (assume for cmd-timezone.feature)
```
$ make integration GODOG_OPTS="-paths ~/go/src/github.com/minishift/minishift/test/integration/features/cmd-timezone.feature -tags quick"
[..]
3 scenarios (3 passed)
9 steps (9 passed)
972.582216ms
testing: warning: no tests to run
PASS
ok      github.com/minishift/minishift/test/integration 1.051s [no tests to run]
```